src/main/java/aeneas/models/PuzzleLevel.java

```diff
@@ -11,10 +11,12 @@
   
   public PuzzleLevel(Bullpen bullpen) {
     super(bullpen);
+    board = new PuzzleBoard();
```

Comment:
Ok, should I just add a separate constructor that allows a board to be passed in? or should I completely remove a level's ability to create a board?